src/basilisp/lang/runtime.py

```diff
@@ -1454,20 +1454,23 @@ def partial_f(*inner_args, **inner_kwargs):
 
 
 @functools.singledispatch
-def deref(o, timeout_s=None, timeout_val=None):
+def deref(o, timeout_ms=None, timeout_val=None):
     """Dereference a Deref object and return its contents.
 
-    If o is an object implementing IBlockingDeref and timeout_s and
-    timeout_val are supplied, deref will wait at most timeout_s seconds,
-    returning timeout_val if timeout_s seconds elapse and o has not
+    If o is an object implementing IBlockingDeref and timeout_ms and
+    timeout_val are supplied, deref will wait at most timeout_ms milliseconds,
+    returning timeout_val if timeout_ms milliseconds elapse and o has not
     returned."""
     raise TypeError(f"Object of type {type(o)} cannot be dereferenced")
 
 
 @deref.register(IBlockingDeref)
 def _deref_blocking(
-    o: IBlockingDeref, timeout_s: Optional[float] = None, timeout_val=None
+    o: IBlockingDeref, timeout_ms: Optional[float] = None, timeout_val=None
```

Comment:
```suggestion
    o: IBlockingDeref, timeout_ms: Optional[int] = None, timeout_val=None
```

I think this will be an integer now, right?

Reply:
Updated, sorry I've missed that.